core/common/src/main/java/alluxio/underfs/BaseUnderFileSystem.java

```diff
@@ -62,7 +62,7 @@ public void configureProperties() throws IOException {
 
   @Override
   public OutputStream create(String path) throws IOException {
-    return create(path, CreateOptions.defaults());
+    return create(path, CreateOptions.defaults().setCreateParent(true));
```

Comment:
Should `setCreateParent` be defaulted to true? It seems weird to have an implicit non-default option here.